api/routers/areas.py

```diff
@@ -113,16 +119,65 @@ async def create_area(new_area: AreaConfigDTO, reboot_processor: Optional[bool]
     return next((area for area in get_areas() if area["id"] == area_dict["Id"]), None)
 
 
+def modify_area_all(area_information):
+    """
+    Edits the configuration related to the area "ALL", an area that contains all cameras.
+    """
+    # Doubt: We force the user to send "cameras", since the input must be AreaConfig DTO. However, I don't think
```

Comment:
The user should not send `cameras`. Can we make that field optional?

Reply:
At first I thought the field must be mandatory because cameras are used by various functions, and sending None will throw an exception. However, I think if we set a default "" when no camera is sent, everything will work fine. I will do it.